api/scripts/wazuh-apid.py

```diff
@@ -60,12 +73,20 @@ def set_logging(log_path='logs/api.log', foreground_mode=False, debug_mode='info
     configuration.api_conf.update(configuration.read_yaml_config(config_file=config_file))
     api_conf = configuration.api_conf
     security_conf = configuration.security_conf
-    log_path = api_conf['logs']['path']
 
     # Set up logger
+    log_path = api_conf['logs']['path']
     set_logging(log_path=log_path, debug_mode=api_conf['logs']['level'], foreground_mode=foreground)
     logger = logging.getLogger('wazuh-api')
 
+    # Set memory usage limit
+    max_memory_usage = int(api_conf['access']['max_memory_usage'])
+    if max_memory_usage >= MIN_VALUE_MAX_MEMORY_USAGE:
+        limit_memory(max_memory_usage)
+    else:
```

Comment:
We should allow using the value 0 in X so as not to limit memory usage. 

```suggestion
    elif max_memory_usage != 0:
```